io/net/virt-net/network_virtualization.py

```diff
@@ -350,6 +351,21 @@ def test_backingdevremove(self):
                            self.backingdev_count)
             self.fail("Failed to remove backing device")
 
+    def test_vnic_dlpar(self):
+        '''
+        Perform vNIC device hot add and hot remove using drmgr command
+        '''
+        self.update_backing_devices()
+        dev_id = self.find_device_id()
+        slot = self.find_virtual_slot(dev_id)
+        for _ in range(self.num_of_dlpar):
+            self.drmgr_vnic_dlpar('-r', slot)
+            time.sleep(5)
```

Comment:
can we find an alternative for this sleep ?

Reply:
How about this ?
wait.wait_for(self.drmgr_vnic_dlpar('-r', slot))

Reply:
Does that work ?